docs/journalist/starting_client.rst

```diff
@@ -1,16 +1,66 @@
-Starting the SecureDrop client
+Starting the SecureDrop Client
 ==============================
 
 .. include:: ../includes/top-warning.rst
 
+Double-click on the Desktop shortcut to launch the SecureDrop Client
+application.
+
+.. | screenshot_desktop-shortcut|
+
 Performing updates
 ------------------
 
+When you launch the application, it will first check for important security
+updates. This can take several minutes.
+
+.. | screenshot_check_updates |
+
+If updates are found, you will be asked to apply them before launching the
+client. You have the option of completing this now, or at another time. Keep in
+mind that updates are important to security, and you should apply them the same
+day you receive the notification.
+
+It is important that, when you do click "Apply Updates," you allow the update
+process to complete fully, without closing or interrupting it. The update
+process may take some time.
+
+.. | screenshot_apply_updates |
+
+Once the update process has been completed, the SecureDrop client will launch.
+
 Signing in
 ----------
 
+To sign in, enter the username and passphrase provided to you by your
+SecureDrop administrator, as well as the 2-factor method you have set up (such
+as Google Authenticator or FreeOTP). If you have used SecureDrop before, these
```

Comment:
Let's use the terminology established here:
https://github.com/freedomofpress/securedrop/pull/5014

So I would write "as well as the two-factor code using the method you have set up"

Reply:
fixed in 437ad10